wicketd/src/update_tracker.rs

```diff
@@ -210,24 +224,25 @@ impl UpdateTracker {
                 )),
             };
 
-            let task = tokio::spawn(update_driver.run(plan));
+            let task =
+                tokio::spawn(update_driver.run(plan, ipr_start_receiver));
 
-            SpUpdateData { task, update_log }
+            Ok::<_, StartUpdateError>(SpUpdateData { task, update_log })
```

Comment:
Why is this closure returning a result now? It doesn't look like the addition of `ipr_start_receiver` adds any fallible calls - maybe it did at one point?

Reply:
Ah yeah, it used to -- will fix.